corehq/apps/reports/standard/cases/basic.py

```diff
@@ -265,6 +266,22 @@ class CaseListReport(CaseListMixin, ProjectInspectionReport, ReportDataSource):
     name = ugettext_lazy('Case List')
     slug = 'case_list'
 
+    @classmethod
+    def get_subpages(cls):
+        def _case_name(request=None, **context):
+            if 'case' in context:
+                return mark_safe(context['case'].name)
+            else:
+                return _('View Case')
+
+        from corehq.apps.reports.views import CaseDataView
+        return [
+            {
+                'title': _case_name,
```

Comment:
nit: a key like `'get_title'` would have made it more clear that the value is a function. I was confused at first when reading the code.

Aside: it's unfortunate but there are other places in the hq codebase where nouns are used for function names and verb-ish phrases (like `get_id`) are property names.

P.S. Sorry, I should have added this comment before merging.